src/renderer/src/components/model-selector.tsx

```
@@ -0,0 +1,8 @@
+// lets you select the various models from all the available providers
+function ModelSelector() {
```

ESLint

[error] Fast refresh only works when a file has exports. Move your component(s) to a separate file.

(react-refresh/only-export-components)